openedx/core/lib/request_utils.py

```diff
@@ -207,94 +205,94 @@ def _get_expected_error_settings_dict():
                 'is_ignored': True,
                 'log_error': True,
                 'log_stack_trace': True,
-                'reason_expected': 'In most cases, signifies a user was trying to do something they cannot do. '
+                'reason_ignored': 'In most cases, signifies a user was trying to do something they cannot do. '
                    'However, an overabundance could indicate a bug, which could be monitored for.'
             }
             ...
         }
 
     """
-    global _EXPECTED_ERROR_SETTINGS_DICT
+    global _IGNORED_ERROR_SETTINGS_DICT
 
     # Return cached processed mappings if already processed
-    if _EXPECTED_ERROR_SETTINGS_DICT is not None:
-        return _EXPECTED_ERROR_SETTINGS_DICT
+    if _IGNORED_ERROR_SETTINGS_DICT is not None:
+        return _IGNORED_ERROR_SETTINGS_DICT
 
-    expected_errors = getattr(settings, 'EXPECTED_ERRORS', None)
-    if expected_errors is None:
-        _EXPECTED_ERROR_SETTINGS_DICT = {}
-        return _EXPECTED_ERROR_SETTINGS_DICT
+    ignored_errors = getattr(settings, 'IGNORED_ERRORS', None)
+    if ignored_errors is None:
+        _IGNORED_ERROR_SETTINGS_DICT = {}
+        return _IGNORED_ERROR_SETTINGS_DICT
 
     # Use temporary variable to build mappings to avoid multi-threading issue with a partially
     # processed map.  Worst case, it is processed more than once at start-up.
-    expected_error_settings_dict = {}
+    ignored_error_settings_dict = {}
 
     try:
-        for index, expected_error in enumerate(expected_errors):
-            module_and_class = expected_error.get('MODULE_AND_CLASS')
-            processed_expected_error = {
-                'is_ignored': expected_error.get('IS_IGNORED', True),
-                'log_error': expected_error.get('LOG_ERROR', False),
-                'log_stack_trace': expected_error.get('LOG_STACK_TRACE', False),
-                'reason_expected': expected_error.get('REASON_EXPECTED'),
+        for index, ignored_error in enumerate(ignored_errors):
+            module_and_class = ignored_error.get('MODULE_AND_CLASS')
+            processed_ignored_error = {
+                'is_ignored': ignored_error.get('IS_IGNORED', True),
```

Comment:
This should be removed. We no longer need this variable because it is always True.

Reply:
@robrap I think I already did it not missed.[check here](https://github.com/openedx/edx-platform/pull/33184/commits/6357bb0a1617aa34a640797bd7f646cc8b353c36)